projects/angular/src/lib/maskito.directive.ts

```diff
@@ -55,6 +55,7 @@ export class MaskitoDirective implements OnDestroy, OnChanges {
         }
 
         ngZone.runOutsideAngular(() => {
+            this.maskedElement?.destroy();
```

Comment:
Let's revert this line and extend this condition instead 
https://github.com/taiga-family/maskito/blob/691b4eb84eeaee45f4b9e0be834e1100e1c0834b/projects/angular/src/lib/maskito.directive.ts#L51-L55

⬇️ 
```ts
if (
  this.elementPredicate !== elementPredicate ||
  this.options !== options
) {
  // Ignore the result of the predicate if the
  // maskito element (or its options) has changed before the predicate was resolved.
  return;
}
```

Reply:
@TheHomeGera Hello!
Could you update your PR according to this thread ?